### PR TITLE
added basic admin page + modifications to nav

### PR DESCRIPTION
**Pull request by anniewtang**
- added link to nav bar for admin page if username matches a certain string
- currently only with annie’s hardcoded username
- will update in the future with more sustainable/adaptable code

### CODE REVIEW

**Comment by anniewtang** [comment]
fixed merge conflicts with activity feed links

**Comment by anniewtang** [comment]
added chibuikem's username to the permissions

**Comment by adrnswanberg** [comment]
Hey Annie,

Please revert your MVP changes in this branch so we can review the prototype separately :)

**Comment by adrnswanberg** [comment]
Ah wait, nevermind. I don't see your MVP changes in here.

**Comment by anniewtang** [comment]
I think potentially my last commit "added a admin page servlet test file" has things from the MVP — should I revert that commit?

File: src/main/java/codeu/controller/AdminServlet.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        List<Message> listOfAllMessages = getAllMessages(listOfAllConversations);
+        request.setAttribute("messages", listOfAllMessages);
+
+        List<User> listOfAllUsers = userStore.getAllUsers();
+        request.setAttribute("users", listOfAllUsers);
+
+        request.getRequestDispatcher("/WEB-INF/view/admin.jsp").forward(request, response);
+  }
+
+  // @Override
```
Comment (adrnswanberg) [inline src/main/java/codeu/controller/AdminServlet.java]: Remove commented out code before submitting

File: src/test/java/codeu/controller/AdminPageServletTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class AdminPageServletTest {
+
+  private ChatServlet adminServlet;
```
Comment (adrnswanberg) [inline src/test/java/codeu/controller/AdminPageServletTest.java]: should this be AdminServlet?

File: src/test/java/codeu/controller/AdminPageServletTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    mockSession = Mockito.mock(HttpSession.class);
+    Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
+
+    mockResponse = Mockito.mock(HttpServletResponse.class);
+    mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);
+    Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/chat.jsp"))
+        .thenReturn(mockRequestDispatcher);
+
+    mockConversationStore = Mockito.mock(ConversationStore.class);
+    chatServlet.setConversationStore(mockConversationStore);
```
Comment (adrnswanberg) [inline src/test/java/codeu/controller/AdminPageServletTest.java]: adminServlet? here and below.

File: src/test/java/codeu/controller/AdminPageServletTest.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    mockMessageStore = Mockito.mock(MessageStore.class);
+    chatServlet.setMessageStore(mockMessageStore);
+
+    mockUserStore = Mockito.mock(UserStore.class);
+    chatServlet.setUserStore(mockUserStore);
+  }
+
+  @Test
+  public void testDoGet() throws IOException, ServletException {
+    adminServlet.doGet(mockRequest, mockResponse);
```
Comment (adrnswanberg) [inline src/test/java/codeu/controller/AdminPageServletTest.java]: You probably want to test that the servlet returns whatever values the mock stores are configured to return, but that can wait for the next PR.

**Review by adrnswanberg — CHANGES_REQUESTED** [review]

File: src/test/java/codeu/controller/AdminPageServletTest.java (hunk 2 above)
Comment (anniewtang) [reply src/test/java/codeu/controller/AdminPageServletTest.java]: yeah this was a commit i made before actually writing the test file! i meant to revert this commit too but i guess i didn’t o:

Comment (anniewtang) [reply src/test/java/codeu/controller/AdminPageServletTest.java]: yeah this was a commit i made before actually writing the test file! i meant to revert this commit too but i guess i didn’t o:

File: src/test/java/codeu/controller/AdminPageServletTest.java (hunk 4 above)
Comment (anniewtang) [reply src/test/java/codeu/controller/AdminPageServletTest.java]: for the admin-page-mvp branch i think things should’ve been fixed!

Comment (anniewtang) [reply src/test/java/codeu/controller/AdminPageServletTest.java]: for the admin-page-mvp branch i think things should’ve been fixed!

**Review by anniewtang — COMMENTED** [review]
Deleted  commented code & reverted files back to prototype only

**Review by adrnswanberg — APPROVED** [review]

**Comment by adrnswanberg** [comment]
Make sure to merge this before merging your MVP pull request!

**Pull request merged**